hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -1006,6 +1056,41 @@ u32 _imxrt_getStateCM4(void)
 }
 
 
+int _imxrt_wdgInfo(_imxrt_wdgInfo_t *info)
+{
+	u32 val;
+	int res;
+
+	info->enabled = ((*(imxrt_common.wdog1 + wdog_wcr) & (1u << 2)) != 0u) ? 1u : 0u;
+	info->timeoutMs = ((u32)(*(imxrt_common.wdog1 + wdog_wcr) >> 8) * 500u) + 500u;
```

Comment:
Nitpick - two accesses to the register, might be better to use temp variable instead